### DISCUSSION TITLE
b-table filtering by each cell separately and not by the row as a whole

### DISCUSSION THREAD

**Discussion by JuSik404** [bootstrap-vue/bootstrap-vue#6806] [Q&A]
For example, I have fields: _Name, City, Descrtiption._
Items: `[
{'Bob', 'Paris', 'Mother Mary Paris' },
{'Mary', 'Paris', 'Son Bob Paris' },
]`

If i filter by **'Mary Paris'**, the result will be 2 items, but I need one, where the _Name = 'Bob'_
How to avoid row stringification to one common line or make the line separator not a space, but another rarely used character?

I do not want to restrict the search in the table to only one column.

Have any ideas?

**Comment by lichaozhy** [comment]
- Using provider functions, filtering by backend.
  https://bootstrap-vue.org/docs/components/table#using-items-provider-functions

- Custom filter function to implement your feature
  https://bootstrap-vue.org/docs/components/table#custom-filter-function

- Setting the `items` of a &lt;b-table> from a vue "computed property" which returns a data array with your owned filter logic.
